index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio skeleton</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+  <main>
+
+    <div class="container"></div>
+      <header>
+        <nav class="navbar">
+          <div class="header-top-art">
+            <img src="./images/image_geometry_1.png" alt="top left art"/>
```

Comment:
- [ ] You should add this image as `background image` for the header section using CSS background-image because the image is a decorator image and not part of the HTML content. Please check this [link](https://www.w3schools.com/cssref/pr_background-image.asp) for how to use background images in CSS.

Reply:
.header-top-art {

  justify-content:flex-start;
  padding: 0px;
  background-image: url("image_geometry1.png") ;

  i must to delete the <img src= >  from html ?